types-of-wireless-card/m2.md

```diff
@@ -23,7 +23,7 @@ Asus and Lenovo users should also see the [Bluetooth](/misc/bluetooth.md) sectio
 ## Supported Cards
 
 * **BCM94360NG**:
-  * Fenvi BCM94360NG(A+E Key, natively supported as based off of genuine Apple Airport card)(BT 4.0)
+  * Fenvi BCM94360NG(A+E Key, natively supported as based off of genuine Apple Airport card)(BT 4.0) (There are confirmed reports that available drivers for this card either do not work or crash the OS on Windows 11, on at least some systems, so while fully compatible with macOS this card may not be suitable for those wishing to dual-boot)
```

Comment:
This is a bit hard to read. Can you either split it into two sentences or rearrange it a bit?

For example:
`...or crash Windows 11. While fully compatible with macOS, this card...`

Reply:
Resolved, I hope. Thanks, sorry for the delay!